### ISSUE TITLE
estimation fails with stocktd 

### ISSUE THREAD

**Issue by annasmyk** [palatej/rjd3toolkit#28]
estimation fails with stocktd when a rsa3 spec is customized with set trading days 
(works if a rsa5c spec is customized)

y_raw<-rjd3toolkit::ABS$X0.2.08.10.M
spec_x13_d<-rjd3x13::spec_x13("rsa3")
spec_x13_d<- set_tradingdays(spec_x13_d,stocktd=28)
spec_x13_d$regarima$regression$td # w=28: ok 
sa_x13_d<- rjd3x13::x13(y_raw, spec_x13_d)
returns 
Error in .jcall("jdplus/x13/base/r/X13", "Ljdplus/x13/base/api/x13/X13Spec;", :
java.lang.NullPointerException: tdc is marked non-null but is null

**Comment by AQLT** [comment]
I think this is a `rjd3x13` issue and not `rjd3toolkit`. The error occurs when the test is NO:
```
y_raw<-rjd3toolkit::ABS$X0.2.08.10.M
spec_x13_d<-rjd3x13::spec_x13("rsa3") |> set_tradingdays(stocktd=28)
spec_x13_d$regarima$regression$td$test = "REMOVE"
sa_x13_d<- rjd3x13::x13(y_raw, spec_x13_d) # works
spec_x13_d$regarima$regression$td$test = "ADD"
sa_x13_d<- rjd3x13::x13(y_raw, spec_x13_d) # works
spec_x13_d$regarima$regression$td$test = "NO"
sa_x13_d<- rjd3x13::x13(y_raw, spec_x13_d) # doesn't work
```

**Comment by palatej** [comment]
An old useless test was generating the problem
Corrected in 3.0.2.
